addins/Cake.XmlDocMarkdown.yml

```diff
@@ -2,7 +2,7 @@ Name: Cake.XmlDocMarkdown
 NuGet: Cake.XmlDocMarkdown
 Assemblies:
 - "/**/Cake.XmlDocMarkdown.dll"
-Repository: https://github.com/ejball/XmlDocMarkdown
+Repository: https://ejball.com/XmlDocMarkdown
```

Comment:
@Jericho similar question to above.

Reply:
The author has specifically used https://ejball.com/XmlDocMarkdown in their nuspec. See the "project site" link in this page for evidence : https://www.nuget.org/packages/Cake.XmlDocMarkdown/